jwst/regtest/conftest.py

```diff
@@ -193,55 +194,63 @@ def rtdata_module(artifactory_repos, envopt, request):
 
 @pytest.fixture
 def fitsdiff_default_kwargs():
+    ignore_keywords = ['DATE', 'CAL_VER', 'CAL_VCS', 'CRDS_VER', 'CRDS_CTX',
+        'NAXIS1', 'TFORM*']
     return dict(
         ignore_hdus=['ASDF'],
-        ignore_keywords=['DATE','CAL_VER','CAL_VCS','CRDS_VER','CRDS_CTX'],
-        ignore_fields=['DATE','CAL_VER','CAL_VCS','CRDS_VER','CRDS_CTX'],
-        rtol=0.00001,
-        atol=0.0000001,
+        ignore_keywords=ignore_keywords,
+        ignore_fields=ignore_keywords,
+        rtol=1e-5,
+        atol=1e-7,
     )
 
 
 @pytest.fixture
 def diff_astropy_tables():
     """Compare astropy tables with tolerances for float columns."""
 
-    def _diff_astropy_tables(actual_path, truth_path, rtol=0.00001, atol=0.0000001):
-        actual = Table.read(actual_path)
+    def _diff_astropy_tables(result_path, truth_path, rtol=1e-5, atol=1e-7):
+        result = Table.read(result_path)
         truth = Table.read(truth_path)
 
         diffs = []
 
-        if actual.colnames != truth.colnames:
+        if truth.colnames != truth.colnames:
```

Comment:
Shouldn't this be ``result.colnames`` on the left side?

Reply:
Yes to all of these.  Thanks for catching.  Fixing all of them.

Reply:
Also, `flake8` caught this bug in the CI for this PR:

https://travis-ci.org/spacetelescope/jwst/jobs/640132790#L595